src/window/building/distribution.c

```diff
@@ -103,7 +121,9 @@ static struct {
     int building_id;
     int partial_resource_focus_button_id;
     int tooltip_id;
-} data = {0, 0, 0, 0, 0, 0, 0};
+    int dock_scrollbar_position;
+    int dock_max_cities_visible;
+} data = {0, 0, 0, 0, 0, 0, 0, 0};
```

Comment:
```suggestion
} data;
```

You don't need to set everything to `0`. [`static` C elements already start zeroed](https://stackoverflow.com/questions/3373108/why-are-static-variables-auto-initialized-to-zero).